src/hazelcore/semantics/Action.re

```diff
@@ -6,6 +6,9 @@ type operator_shape =
   | SMinus
   | SPlus
   | STimes
+  | SFMinus
+  | SFPlus
+  | SFTimes
```

Comment:
Do we actually need these? Maybe we'd eventually want to directly construct via keyboard command, but even that seems like overkill to me. cc @cyrus-